src/run.rs

```diff
@@ -47,6 +47,11 @@ pub fn run() {
   #[cfg(windows)]
   enable_ansi_support().ok();
 
+  let invocation_directory = match env::current_dir() {
+    Ok(pathbuf) => pathbuf,
```

Comment:
Let's do `pathbuf.to_str().map(str::to_string)` and then use `Option<String>` for the type of the `invocation_directory` value.